lib/js-transform.js

```diff
@@ -96,3 +136,53 @@ function yamlTransform (filename) {
     next()
   })
 }
+
+const base64ify = postcss.plugin('postcss-base64ify', function () {
+  return function (css, result) {
+    css.replaceValues(/url\((\s*)(['"]?)(.+?)\2(\s*)\)/g, function (string) {
+      const filename = getUrl(string)
+        .split('?')[0]
+        .split('#')[0]
+      let file
+      if (
+        filename.indexOf('data') === 0 ||
```

Comment:
This could be shortened a bit by using `.startsWith('data')`, but it's not crucial (and same below, of course).

Reply:
This function was moved from elsewhere, to change where it's loaded in the pipeline. I'd rather replace this with an `npm` package that does the same than refactor it.

Reply:
Ah okay, yeah, that makes more sense to me now, I thought something like that might have been the case but I didn't see any deletions in the PR for this so I wasn't sure. I agree with replacing it with a package if we get a chance to spend more time on this.

Reply:
:+1: it was in `lib/css-transform.js`, which GitHub truncated in its diff.

Reply:
Ugh, that's my least-favorite GitHub feature; 98% of the time it's from file renames or similar so I tend to skip over them but then every so often it'll truncate something substantive. I'll have to get in the habit of always expanding them because this is not the first time this has happened to me.